### ISSUE TITLE
[Question] Spam userbots solving the captcha?

### ISSUE THREAD

**Issue by shentino** [J-Rios/TLG_JoinCaptchaBot#121]
Just wanted to give a heads up to the devs that I just got hit by a pakistani promotions spambot that successfully completed the captcha.

**Comment by J-Rios** [comment]
Ok, let's keep this issue open to see if there is more users reporting this and keep track of the evolution of userbots that solve TJCB captchas.

**Comment by J-Rios** [comment]
![captcha2](https://user-images.githubusercontent.com/12136967/132831888-feafd191-b917-4d51-8d9e-6ae6c1f98202.jpg)

I have updated [multicolorcaptcha library](https://github.com/J-Rios/multicolorcaptcha) to use new and interesting Fonts.

Hopefully it will help to break spam userbots that are solving the current captchas images...

Will be added for next Bot vrsion update.

**Comment by shentino** [comment]
Yeah from what I remember they do have "captcha solver" sweat shops out there taking advantage of poor people.

Not sure how to tell the difference between them and a well coded AI
